File: source/code/csharp/account_management/create_account_in_dir_req.cs
var picard = client.Instantiate<IAccount>()
    .SetUsername("jlpicard")
    .SetEmail("capt@enterprise.com")
    .SetGivenName("Jean-Luc")
    .SetSurname("Picard")
    .SetPassword("uGhd%a8Kl!");
await captainsDirectory.CreateAccountAsync(picard);

File: source/code/java/account_management/create_account_in_dir_req.java
Account picard = client.instantiate(Account.class)
    .setUsername("jlpicard")
    .setEmail("capt@enterprise.com")
    .setGivenName("Jean-Luc")
    .setSurname("Picard")
    .setPassword("uGhd%a8Kl!");

captainsDirectory.createAccount(picard);
